package.json

```diff
@@ -36,9 +36,15 @@
         "@0x/subproviders": "^6.0.2",
         "@0x/utils": "^5.1.1",
         "@0x/web3-wrapper": "^7.0.2",
-        "@reach/dialog": "^0.1.3",
-        "@reach/tabs": "^0.1.6",
-        "@types/d3": "^5.7.2",
+        "@reach/dialog": "^0.11.2",
+        "@reach/tabs": "^0.11.2",
+        "@web3-react/abstract-connector": "latest",
+        "@web3-react/core": "latest",
+        "@web3-react/injected-connector": "latest",
+        "@web3-react/network-connector": "latest",
+        "@web3-react/types": "latest",
+        "@web3-react/walletconnect-connector": "latest",
+        "@web3-react/walletlink-connector": "latest",
```

Comment:
Can we use the normal versioning scheme here, not `latest`. Otherwise this will definitely break down the line